src/test/java/edu/cornell/kfs/module/purap/iwant/xml/TestIWantXmlObjects.java

```diff
@@ -0,0 +1,100 @@
+package edu.cornell.kfs.module.purap.iwant.xml;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.xmlunit.diff.DifferenceEvaluators;
+
+import edu.cornell.kfs.module.purap.iwant.xml.fixture.IWantDocumentWrapperFixture;
+import edu.cornell.kfs.sys.service.impl.CUMarshalServiceImpl;
+import edu.cornell.kfs.sys.util.CuXMLUnitTestUtils;
+import edu.cornell.kfs.sys.util.KualiDecimalXmlDifferenceEvaluator;
+import jakarta.xml.bind.JAXBException;
+
+@Execution(ExecutionMode.SAME_THREAD)
+public class TestIWantXmlObjects {
```

Comment:
For consistency with our other test classes, I would suggest renaming this class so that it ends with "Test".